Tests/ContextSyncTests.swift

```diff
@@ -118,32 +118,32 @@ final class ContextSyncTests: TestCase {
             companies = self.generateDataInContext(childContext, companiesCount: 3, employeesCount: 3)
         }
         let companyNames = companies.map { $0.name }
-
+        
         // WHEN: we save the child context
-        expectation(forNotification: Notification.Name.NSManagedObjectContextDidSave.rawValue, object: childContext, handler: nil)
+        expectation(forNotification: Notification.Name.NSManagedObjectContextDidSave, object: childContext, handler: nil)
```

Comment:
Could you just use `expectation(forNotification: .NSManagedObjectContextDidSave, object:…` here?